File: app/src/main/java/bj/rxjavaexperimentation/discogs/gson/release/Submitter.java

package bj.rxjavaexperimentation.discogs.gson.release;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Submitter {

    @SerializedName("resource_url")
    @Expose
    private String resourceUrl;
    @SerializedName("username")
    @Expose
    private String username;

    public String getResourceUrl() {
        return resourceUrl;
    }

    public void setResourceUrl(String resourceUrl) {
        this.resourceUrl = resourceUrl;
    }

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

}

File: app/src/main/java/bj/vinylbrowser/model/release/Submitter.kt
package bj.vinylbrowser.model.release

import android.os.Parcel
import android.os.Parcelable
import com.google.gson.annotations.SerializedName

/**
 * Created by Josh Laird on 19/05/2017.
 */
data class Submitter(@SerializedName("resource_url") val resourceUrl: String,
                     val username: String) : Parcelable {
    override fun writeToParcel(dest: Parcel, flags: Int) {
        dest.writeString(resourceUrl)
        dest.writeString(username)
    }

    override fun describeContents(): Int {
        return 0
    }
}